xExtension-YouTube/extension.php

```diff
@@ -176,29 +190,76 @@ public function getIFrameForLink($link)
     * @param string $link
     * @return string
     */
-    public function getPeerTubeIFrameForLink($link)
+    public function getHtmlPeerTubeContentForLink($entry, $link)
     {
         $url = str_replace('/watch', '/embed', $link);
-        $html = $this->getIFrameHtml($url);
+        $html = $this->getHtml($entry, $url);
         
         return $html;
     }
 
     /**
-     * Returns an HTML <iframe> for a given URL for the configured width and height.
+     * Returns an HTML <iframe> for a given URL for the configured width and height, with content ignored, appended or formated.
      *
      * @param string $url
      * @return string
      */
-    public function getIFrameHtml($url)
+    public function getHtml($entry, $url)
     {
-        return '<iframe class="youtube-plugin-video"
+        $content = '[error]';
```

Comment:
(nitpick) What's the purpose of the `[error]` value? `$content` seems to be always overridden, isn't it?

Reply:
Yes, I changed this fact by @kevinpapst suggestion, and forgot to remove this. By the way, what is the practice in this case, in PHP ?

`$content = '';` and let the switch cases override it, or just not define it, and let the switch cases defining it ?

Reply:
I personally prefer to define it with `$content = '';` so I'm sure the variable exists after the `switch`, but I don't know if there's any good PHP practice that defines this ^^

Reply:
Even though it is not really necessary in PHP I would also always define it above the switch, either as null or as empty string if required. The `[error]` will never be used, so it should be changed.

Reply:
Okay, fixed.